usecase/src/test/java/org/mobilitydata/gtfsvalidator/usecase/ValidateStopTimeTimeCombinationTest.java

```diff
@@ -0,0 +1,461 @@
+/*
+ *  Copyright (c) 2020. MobilityData IO.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.usecase;
+
+import org.junit.jupiter.api.Test;
+import org.mobilitydata.gtfsvalidator.domain.entity.gtfs.stoptimes.StopTime;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.error.BadStopTimeTimeCombinationNotice;
+import org.mobilitydata.gtfsvalidator.usecase.port.ValidationResultRepository;
+import org.mobilitydata.gtfsvalidator.usecase.utils.TimeUtils;
+import org.mockito.ArgumentCaptor;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mobilitydata.gtfsvalidator.domain.entity.notice.base.Notice.*;
+import static org.mockito.Mockito.*;
+
+class ValidateStopTimeTimeCombinationTest {
+
+    @Test
+    void validTimeCombinationShouldNotGenerateNotice() {
+        // | tripId | stopSequence | departureTime | arrivalTime |
```

Comment:
💯 This visual representation is very helpful!

Reply:
Actually, looks like you have the arrival and departure times reversed in the header. Same for other tests:

| tripId | stopSequence | arrivalTime | departureTime |
|--------|--------------|---------------|-------------|
| 0      | 2            | 07:10         | 09:20       |
| 0      | 5            | 9:45          | 10:11       |
| 0      | 8            | 11:44         | 12:01       |